### PR TITLE
Fix move the cursor to the point, not the line

### PR DESCRIPTION
**Pull request by SuzumiyaAoba**
Hi!

This package moves the cursor to the line from the history.
Is this the specification which you hoped for?

I think it is better that the cursor is moved to the point, not the line.
Please consider my proposal :smile: 

P.S. Maybe you can try making an option of choosing a point or a line.

### CODE REVIEW

**Comment by SuzumiyaAoba** [comment]
This change is related to 1dc4b20.

**Comment by blue0513** [comment]
@SuzumiyaAoba Thx for the proposal!  
Yeah, as you said, moving to the point is better for users. 

I'll review your PR then should merge!

File: point-history.el [hunk 1]
```diff
@@ -118,7 +118,7 @@ If ONLY-CLOSE is non-nil, it dons not call `pop-to-buffer'."
 (defun point-history--goto (buffer-str pos)
   "Pop to BUFFER-STR and go to POS, then close `point-history-show-buffer'."
   (with-selected-window (get-buffer-window point-history-last-previewed-buffer)
-    (goto-line (line-number-at-pos pos))
+    (goto-char pos)
```
Comment (blue0513) [inline point-history.el:121]: nice :+1:

**Review by blue0513 — CHANGES_REQUESTED** [review]
Plz consider these points:  
1. When you close `point-history-show-buffer` by [`point-history-close`](https://github.com/blue0513/point-history/blob/master/point-history.el#L102-L105), it should call `goto-char`
2. To realize it, you should modify `point-history-last-visited-buffer` to contains `position` instead of `line-number` on [these lines](https://github.com/blue0513/point-history/blob/master/point-history.el#L191-L192)

**Review by blue0513 — APPROVED** [review]

**Comment by blue0513** [comment]
@SuzumiyaAoba LGTM! :+1:
But before merge this PR, you should squash these two commits to one. After that, I'll merge it

**Comment by blue0513** [comment]
@SuzumiyaAoba Thx! Merged!

**Pull request merged**